### PR TITLE
refactor(RouterStore): change default state key to router

### LINKED ISSUE #1241: Change default stateKey for router-store to "router"
## If accepted, I would be willing to submit a PR for this feature

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

The current default for stateKey is "routerReducer"

### PR DESCRIPTION
**Pull request by timdeschryver**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #1241 

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

## Other information
Closes #1241 

BREAKING CHANGE:

The default state key is changed from routerReducer to router

### CODE REVIEW

File: docs/router-store/api.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # API
 
+## StoreRouterConnectingModule.forRoot
```
Comment (brandonroberts) [inline docs/router-store/api.md]: Change to `Configuration Options`

File: docs/router-store/api.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+
+To connect the Angular router module with the NgRx store module, import the store module via
```
Comment (brandonroberts) [inline docs/router-store/api.md]: Remove the `function` that accepts an object. Its going to be removed from the API

File: docs/router-store/api.md (hunk 1 above)
Comment (timdeschryver) [reply docs/router-store/api.md]: Done.

File: docs/router-store/api.md (hunk 2 above)
Comment (timdeschryver) [reply docs/router-store/api.md]: Done.

File: docs/router-store/api.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # API
 
+## Configuration Options
+
+To connect the Angular router module with the NgRx store module, import the store module via
+`StoreRouterConnectingModule.forRoot()`.
+
```
Comment (brandonroberts) [inline docs/router-store/api.md]: What happened to the description for `stateKey`?

Comment (timdeschryver) [reply docs/router-store/api.md]: I'm confused 😅 
Like this you mean?

Comment (brandonroberts) [reply docs/router-store/api.md]: 😴 I must have been looking at a partial change. 👍

Comment (timdeschryver) [reply docs/router-store/api.md]: No, I've recommitted. I must have done something wrong...

**Pull request merged**